wagtailio/areweheadlessyet/migrations/0001_initial.py

```diff
@@ -1,89 +1,36 @@
-# Generated by Django 3.2.8 on 2022-01-19 12:58
```

Comment:
Note: Editing migrations is generally a bad idea, as this won't be run during deployment. If we want this to run, you will need to roll back migrations after deployment, before which the migrations won't actually have run.

Reply:
Do you suggest that I keep the previous migrations and just add another one?

Reply:
Given we're so early in the project, let's keep the history clean and keep it as you have it now (ie 1 new commit and remove the old one). It's just something to be aware of when deploying.